classes/01class/exercises/c01-aws02/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,49 @@
+# C01-AWS02
+
+## Commands Execution Output
+
+- Commands to create the S3 Bucket and to copy the file to the bucket:
+```
+$ aws s3 mb s3://mrcsmonteiro-devops-academy-bucket
+make_bucket: mrcsmonteiro-devops-academy-bucket
+
+$ aws s3 cp file.txt s3://mrcsmonteiro-devops-academy-bucket
+upload: ./file.txt to s3://mrcsmonteiro-devops-academy-bucket/file.txt
+
+$ aws s3 ls s3://mrcsmonteiro-devops-academy-bucket
+2020-06-12 20:30:17          0 file.txt
+```
+
+- Commands to allow the EC2 instance to access the files in S3:
+```
+$ aws ec2 associate-iam-instance-profile --instance-id i-00078c13bff5b2222 --iam-instance-profile Name=S3Access4EC2
```

Comment:
can you include the command to create the instance role(and any other additional recource) as well?

Reply:
Done. I think the PR is looking better now :-)